pkg/popeye.go

```diff
@@ -434,7 +434,13 @@ func (p *Popeye) ensureOutput() error {
 }
 
 func (p *Popeye) fileName() string {
-	return fmt.Sprintf(outFmt, p.factory.Client().ActiveCluster(), time.Now().UnixNano(), p.fileExt())
+	var fileName string
```

Comment:
@yogeek Thank you for this PR! I think that make sense. We could simplify here a bit with something like:

```go
if *p.flags.OutputFile == "" {
  return fmt.Sprintf(outFmt, p.factory.Client().ActiveCluster(), time.Now().UnixNano(), p.fileExt())
}
return fmt.Sprintf(*p.flags.OutputFile)
```

Reply:
Indeed, much simpler 👍